src/role-access/models/allocate-role-navigation-event.enum.ts

```diff
@@ -0,0 +1,6 @@
+export enum AllocateRoleNavigationEvent {
+  BACK,
+  CANCEL,
+  CONTINUE,
+  CONFIRM,
+}
```

Comment:
Minor query - Do we need delete here?

Reply:
Nope I think, as the Delete is in different user journey, but you could put Delete here however the navigation doesn't have Delete state in the single page.

Reply:
ok sure